janusgraph-cql/src/test/resources/cql-read.properties

```diff
@@ -31,6 +31,7 @@ cassandra.input.widerows=true
 # SparkGraphComputer Configuration #
 ####################################
 spark.master=local[4]
+spark.driver.host=127.0.0.1
```

Comment:
@li-boxuan It seems this line should fix the issue.

Reply:
Awesome! Do you want to make it a separate PR, since it's unrelated to HBase?

Reply:
Good idea.